src/lib.rs

```diff
@@ -276,6 +276,25 @@ pub struct ChannelCloseResolvingFees {
     pub sat_per_vbyte: u32,
 }
 
+#[allow(clippy::large_enum_variant)]
+pub enum ActionRequiredItem {
```

Comment:
It's a long name. But hard to shorten, I agree. What do you think about `RequiredAction`?

Reply:
I also thought about that option. It can be misunderstood as an object that represents an action itself instead of something that can be acted upon. We can try to find a better name, but between those two, I prefer my proposal. WDYT?